detail.js

```diff
@@ -85,8 +85,8 @@ var app = new Vue({
                         console.log(data.data);
                         //valid response, sort by start date, most recent first
                         this.sightings = data.data.sort(function(a,b){
-                            return new Date(b.StartDateTime) - new Date(a.StartDateTime);
-                          });
+                            return (a.StartDateTime - b.StartDateTime);
+                          }).reverse();
```

Comment:
For some reason my brain is saying this is the same thing as the original sort function. Do you have an example detail page where it was loading in the wrong order so I can work thru an example where the original sort doesn't work? I was using queenie as my example manta ray, and it works for that one.

the original sort was b-a, and the new one is a-b, then reverse it.

Reply:
I thought that b-a would work as well but when I tried it, it didn't. I also tried by converting the string to a Date as you did, and that didn't work for me either. I used #9 Big Bertha as my test but also just used Queenie and it's always in ascending order. I just updated the website with the original code to double check, and it's in ascending order.